specification/cognitiveservices/data-plane/Language/preview/2022-07-01-preview/analyzeconversations-authoring.json

```diff
@@ -0,0 +1,3102 @@
+          "type": "number"
+        }
+      },
+      "x-ms-client-name": "ProjectSettings"
+    },
+    "ConversationalAnalysisAuthoringProjectsMetadata": {
+      "description": "Represents a list of retrieved projects' metadata.",
+      "required": [
+        "value",
+        "nextLink"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The list of projects.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConversationalAnalysisAuthoringProjectMetadata"
+          }
+        },
+        "nextLink": {
+          "description": "The next page link.",
+          "type": "string",
+          "x-nullable": true
+        }
+      },
+      "x-ms-client-name": "ProjectsMetadata"
+    },
+    "ConversationalAnalysisAuthoringProjectTrainedModel": {
+      "description": "Represents a trained model.",
+      "required": [
+        "label",
+        "modelId",
+        "lastTrainedDateTime",
+        "lastTrainingDurationInSeconds",
+        "modelExpirationDate",
+        "modelTrainingConfigVersion"
+      ],
+      "type": "object",
+      "properties": {
+        "label": {
+          "description": "The trained model label.",
+          "type": "string"
+        },
+        "modelId": {
+          "description": "The model ID.",
+          "type": "string"
+        },
+        "lastTrainedDateTime": {
+          "format": "date-time",
+          "description": "The last trained date time of the model.",
+          "type": "string"
+        },
+        "lastTrainingDurationInSeconds": {
+          "format": "int32",
+          "description": "The duration of the model's last training request in seconds.",
+          "type": "integer"
+        },
+        "modelExpirationDate": {
+          "format": "date",
+          "description": "The model expiration date.",
+          "type": "string"
+        },
+        "modelTrainingConfigVersion": {
+          "description": "The model training config version.",
+          "type": "string"
+        }
+      },
+      "x-ms-client-name": "ProjectTrainedModel"
+    },
+    "ConversationalAnalysisAuthoringProjectTrainedModels": {
+      "description": "Represents a list of retrieved trained models.",
+      "required": [
+        "value",
+        "nextLink"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The list of retrieved jobs.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConversationalAnalysisAuthoringProjectTrainedModel"
+          }
+        },
+        "nextLink": {
+          "description": "The next page link.",
+          "type": "string",
+          "x-nullable": true
+        }
+      },
+      "x-ms-client-name": "ProjectTrainedModels"
+    },
+    "ConversationalAnalysisAuthoringStringIndexType": {
```

Comment:
This conflicts with the common StringIndexType. We should use the common object here.